hist/hist/src/TF1.cxx

```diff
@@ -536,7 +534,7 @@ TF1::TF1(const char *name, Double_t (*fcn)(Double_t *, Double_t *), Double_t xmi
    fParent(0), fHistogram(0),
    fMethodCall(0),
    fNormalized(false), fNormIntegral(0),
-   fFunctor(ROOT::Math::ParamFunctor(fcn)),
+   fFunctor(new TF1FunctorPointerImpl<double>(ROOT::Math::ParamFunctor(fcn))),
```

Comment:
I assume `fFunctor` takes the ownership here. Could we add a clarification comment?